src/components/procedure/_enable-turbo-mode.md

```diff
@@ -1,9 +1,9 @@
-:::tip Try Turbo mode (beta)
-Semgrep can automatically scan for findings using the Semgrep OSS Engine, on any change in the Rule or Sample Code panes.
+The **Turbo** option runs your rule automatically against the Semgrep OSS Engine after every keystroke or change to the rule.
 
-Click the <i class="fa-solid fa-toggle-large-on"></i> **Turbo mode** toggle to try it out.
+Click the <i class="fa-solid fa-toggle-large-on"></i> **Turbo mode** toggle to enable.
 
-In this mode, the **Run** button is hidden because Semgrep scans upon detecting any change to the text. Turbo mode works in both Simple mode and Advanced mode. Turbo mode cannot be enabled for Semgrep Pro Engine.
+The **Run** button is hidden in Turbo mode because Semgrep scans after any changes to the rule. Turbo mode works with the editor in either simple or advanced mode.
```

Comment:
```suggestion
The **Run** button is hidden in Turbo mode because Semgrep scans after any changes to the rule. Turbo mode works with the editor in either structure or advanced mode.
```

Reply:
(simple mode won't actually exist anymore)